pyproject.toml

```
@@ -1,31 +1,36 @@
 [build-system]
-requires = ["poetry-core", "ziggy-pydust==0.23.2"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling"]
```

maybe I'm missing something, but shouldn't this also include `ziggy-pydust`?

```suggestion
requires = ["hatchling", "ziggy-pydust==0.23.2"]
```